lib/util.js

```diff
@@ -103,27 +87,9 @@ export function publicKeyDerEncode({publicKeyBytes}) {
   // add a zero byte to the front of the publicKeyBytes, this results in
   // the bitstring being 256 bits vs. 170 bits (without padding)
   const zeroBuffer = Buffer.from(new Uint8Array([0]));
-  const keyBuffer = new ByteBuffer(Buffer.concat([zeroBuffer, publicKeyBytes]));
-
-  const a = asn1.create(
-    asn1.Class.UNIVERSAL,
-    asn1.Type.SEQUENCE,
-    true, [
-      asn1.create(asn1.Class.UNIVERSAL, asn1.Type.SEQUENCE, true, [
-        asn1.create(
-          asn1.Class.UNIVERSAL,
-          asn1.Type.OID,
-          false,
-          asn1.oidToDer(oids.EdDSA25519).getBytes()
-        ),
-      ]),
-      // public key
-      asn1.create(
-        asn1.Class.UNIVERSAL, asn1.Type.BITSTRING, false,
-        keyBuffer.getBytes()),
-    ]
-  );
-
-  const publicKeyDer = asn1.toDer(a);
-  return Buffer.from(publicKeyDer.getBytes(), 'binary');
+  const keyBuffer = Buffer.concat([zeroBuffer, publicKeyBytes]);
+  const DERBuffer = Buffer.from([
```

Comment:
This should be a single `DER_PUBLIC_KEY_PREFIX` constant at the top of the file.

Reply:
Addressed here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/5/commits/83c242815aeff1e173bc9c1ca4bf6bdb4f38ad50